lynx/common/enitity.py

```diff
@@ -9,20 +10,24 @@ class Entity(Serializable):
     @dataclass
     class Exported(Serializable):
```

Comment:
`Exported` should not be exported I think :D. I mean that it's public class variable of `Entity` i.e `Entity.Exported`. I'd make it private but please correct me if I miss something. 

I also wanted to point that the name `Exported` is quite bad, but seeing it in a form of `Entity.Exported` changed my mind :)

Reply:
I'm a bit confused when I see a class definition nested in another class. It seems to me that this unnecessarily increases code complexity, but I may be mistaken, so I'm open to any feedback :)

Reply:
@blazej-smorawski I will rename it to _Exported.
@Krystian030 This is a common practice in Python when you want to make sure that the nested class is always used in tandem with the class in which it is nested.

Reply:
@filip-szweda, ok, thanks for the explanation